scripts/parser-tests/typescript/allowlist.txt

```diff
@@ -1,143 +1,5797 @@
-# 7 invalid programs did not produce a parsing error
+# 0 invalid programs did not produce a parsing error
 
-collisionArgumentsArrowFunctions.ts # TypeScript doesn't allow a parameter to be named arguments even in non-strict mode, which we don't catch.
-collisionArgumentsFunction.ts # TypeScript doesn't allow a parameter to be named arguments even in non-strict mode, which we don't catch.
-collisionArgumentsFunctionExpressions.ts # TypeScript doesn't allow a parameter to be named arguments even in non-strict mode, which we don't catch.
-dynamicImportTrailingComma.ts
-exportDeclarationsInAmbientNamespaces2.ts
-multipleExports.ts
-regularExpressionAnnexB.ts # Babel parser does not handle regex parsing
 
 
-# 130 valid programs produced a parsing error
 
+# 5790 valid programs produced a parsing error
```

Comment:
This one seems fishy: It seems to me that the third-party parser testing completely fails so every valid program now produces an error: maybe because the parser is now an ESM build or is there any misconfugrations in the ts parser tests?